round.go

```diff
@@ -16,10 +16,10 @@ package apd
 
 import "math/big"
 
-// Round sets d to rounded x. If d has zero Precision, no rounding will
-// occur. If d has no Rounding specified, RoundHalfUp is used.
+// Round sets d to rounded x. If c has zero Precision, no rounding will
```

Comment:
"Round sets d to rounded x, rounded to the precision specified by c." would be better.